rubygem-foreman_expire_hosts/foreman_expire_hosts.spec

```diff
@@ -0,0 +1,95 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_expire_hosts
+
+Summary:    A Foreman plugin to allow hosts to expire.
+Name:       %{?scl_prefix}rubygem-%{gem_name}
+Version:    2.0.0
+Release:    1%{?foremandist}%{?dist}
+Group:      Applications/System
+License:    GPLv3
+URL:        https://github.com/theforeman/foreman_expire_hosts
+Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+Source1:    %{gem_name}.cron.d
```

Comment:
Also it'd be useful to include this in the repo and gem so it's shared between RPMs/debs/other users.